vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/registration/batch/SampleRegistrationSpreadsheet.java

```diff
@@ -474,10 +472,12 @@ public Result<Void, InvalidSpreadsheetInput> areInputsValid() {
       // mandatory not filled in --> invalid
       for (int colId : mandatoryInputCols) {
         Cell cell = row.getCell(colId);
-        if (SpreadsheetMethods.cellToStringOrNull(cell).isBlank()) {
-          invalidCells.add(cell);
-        } else {
+
+        if (isCellValueValid(cell)) {
           validCells.add(cell);
```

Comment:
We have to be careful with the two PRs here, the other PR changes the following:

          // if a background color was set, but the cell is valid, we need to change the style
          if(cell.getCellStyle().getFillBackgroundColorColor()!=null) {
            validCells.add(cell);
          }

Reply:
just fyi

Reply:
I think I resolved it